### PR TITLE
Added support for Debian 9

### PR DESCRIPTION
**Pull request by harnas-google**

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -15,45 +15,88 @@
...
       - '8.0.3'
       - '8.0'
       - 'latest'
-    from: *from
+    from: *from9
     packages:
       gosu: *gosu
       mysql:
-        version: 8.0.3-rc-1debian8
+        version: 8.0.11-1debian9
```
Comment (jprzychodzen) [inline versions.yaml]: Version mismatch, 8.0.3 in tag vs 8.0.11 in package

Comment (jprzychodzen) [reply versions.yaml]: Also, there is no 8.0.11 in Debian 8.

File: templates/Dockerfile.template [hunk 2]
```diff
@@ -122,4 +132,4 @@ RUN ln -s usr/local/bin/docker-entrypoint.sh /entrypoint.sh  # for backwards com
 ENTRYPOINT ["docker-entrypoint.sh"]
 
 EXPOSE 3306
-CMD ["mysqld"]
+CMD ["mysqld"]
```
Comment (jprzychodzen) [inline templates/Dockerfile.template]: No new line, please add.

File: templates/Dockerfile.template [hunk 3]
```diff
@@ -1,117 +1,127 @@
...
 # File::Basename
 # File::Copy
 # Sys::Hostname
 # Data::Dumper
-		perl \
-	&& rm -rf /var/lib/apt/lists/*
-{{- else if (eq .Dir "8/8.0") }}
+    perl \
+  && rm -rf /var/lib/apt/lists/*
+{{- else if (eq $mysql.Major "8.0") }}
```
Comment (jprzychodzen) [inline templates/Dockerfile.template]: Very nice change! Can we make it future proof, with global variable checking if version is not 5.7 and not 5.6? This way if 8.1 will happen, we will not have a breaking change.

What do you think about that?

File: templates/Dockerfile.template [hunk 4]
```diff
@@ -1,117 +1,127 @@
...
-		echo mysql-community-server mysql-community-server/remove-test-db select false; \
-	} | debconf-set-selections \
-{{- if eq .Dir "8/8.0" }}
-	&& apt-get update && apt-get install -y mysql-community-client-core="${MYSQL_VERSION}" mysql-community-server-core="${MYSQL_VERSION}" && rm -rf /var/lib/apt/lists/* \
+    echo mysql-community-server mysql-community-server/data-dir select ''; \
+    echo mysql-community-server mysql-community-server/root-pass password ''; \
+    echo mysql-community-server mysql-community-server/re-root-pass password ''; \
+    echo mysql-community-server mysql-community-server/remove-test-db select false; \
+  } | debconf-set-selections \
+{{- if eq $mysql.Major "8.0" }}
```
Comment (jprzychodzen) [inline templates/Dockerfile.template]: I think this should be also covered by global variable "not 5.6 and not 5.7".

File: versions.yaml (hunk 1 above)
Comment (harnas-google) [reply versions.yaml]: Tags in version for Debian 9 - done
I do not touch version for Debian 8, this commit is only adding support for Debian 9

File: templates/Dockerfile.template (hunk 2 above)
Comment (harnas-google) [reply templates/Dockerfile.template]: done

File: versions.yaml (hunk 1 above)
Comment (harnas-google) [reply versions.yaml]: The old directories have been deleted.

**Pull request merged**